### PR TITLE
[12.0][FIX]always reset sequence on creation of stock moves

### LINKED ISSUE #631: stock moves sequence-error created from other modules
module: stock_picking_line_sequence
version: 12.0
**Current behavior**
This module does not integrate with other modules such as (sales, purchase )

for example the stock move sequence created from a" sale order" is not correct, instead of (1,2,3,4....etc) you will find (9,999)for all stock move lines?.
also the stock move sequence created from a" purchase order" also is not correct, you will find lines with a sequence step equal to 5 such as (5,10,15,20,25,......etc.)?

for your information I tested the module with a fresh database but the same error.
could you please check that!

### PR DESCRIPTION
**Pull request by AaronHForgeFlow**
Fixes https://github.com/OCA/stock-logistics-workflow/issues/631

### CODE REVIEW

**Comment by AaronHForgeFlow** [comment]
Basically, the issue comes when stock moves are created individually and the picking is created later. For example, the standard procedure when confirming sales orders.

**Comment by LoisRForgeFlow** [comment]
cc @mahmoudmk1 can you review that your problem is fixed?

**Comment by AaronHForgeFlow** [comment]
Tests are failing, will fix that.

**Comment by AaronHForgeFlow** [comment]
Travis error unrelated: fixed here: https://github.com/OCA/stock-logistics-workflow/pull/633

**Comment by AaronHForgeFlow** [comment]
It seems the error in a test of module `product_cost_price_avco_sync` sometimes arises and sometimes not, but seems unrelated to this module.

**Comment by AaronHForgeFlow** [comment]
cc @mahmoudmk1 can you review that your problem is fixed?

File: stock_picking_line_sequence/views/stock_view.xml [hunk 1]
```diff
@@ -15,13 +15,6 @@
                     <field name="sequence" widget="handle"/>
                     <field name="sequence2"/>
                 </xpath>
-                <xpath expr="//notebook/page/field[@name='move_ids_without_package']" position="attributes">
-                    <attribute name="context">{'picking_type_code': picking_type_code, 'default_picking_id': id,
```
Comment (rousseldenis) [inline stock_picking_line_sequence/views/stock_view.xml:19]: Why removing this ?

Comment (AaronHForgeFlow) [reply stock_picking_line_sequence/views/stock_view.xml:19]: It is defaulting the sequence on creation, and it seems not needed anymore.

**Comment by rousseldenis** [comment]
@AaronHForgeFlow Could you rebase ?

**Review by rousseldenis — APPROVED** [review]

**Comment by rousseldenis** [comment]
@mahmoudmk1

**Pull request merged**